modular_zubbers/code/modules/clothing/suits/jacket.dm

```diff
@@ -63,3 +63,130 @@
 	worn_icon = 'modular_zubbers/icons/donator/skyymed_worn.dmi'
 	armor_type = /datum/armor/toggle_labcoat
 	toggle_noun = "zipper"
+
+/obj/item/clothing/suit/jacket/tailcoat //parent type
+	name = "tailcoat"
+	desc = "A coat usually worn by bunny themed waiters and the like."
+	worn_icon = 'modular_zubbers/icons/mob/clothing/suits/jacket.dmi'
+	icon = 'modular_zubbers/icons/obj/clothing/suits/jacket.dmi'
+	icon_state = "tailcoat"
+	greyscale_colors = "#39393f"
+	greyscale_config = /datum/greyscale_config/tailcoat
+	greyscale_config_worn = /datum/greyscale_config/tailcoat_worn
+	flags_1 = IS_PLAYER_COLORABLE_1
+
+/obj/item/clothing/suit/jacket/tailcoat/bartender
+	name = "bartender's tailcoat"
+	desc = "A coat usually worn by bunny themed bartenders. It has an interior holster for firearms and some extra padding for minor protection."
+	icon_state = "tailcoat_bar"
+	greyscale_colors = "#39393f#ffffff"
+	greyscale_config = /datum/greyscale_config/tailcoat_bar
+	greyscale_config_worn = /datum/greyscale_config/tailcoat_bar_worn
+	armor_type = /datum/armor/tailcoat_bar
+
+/datum/armor/tailcoat_bar
+	melee = 15
+	bullet = 10
+	laser = 10
+	energy = 15
+	fire = 20
+	bomb = 10
+	acid = 20
+	wound = 5
```

Comment:
Ditto, just use the armor vest one since bartenders get a vest (`/datum/armor/suit_armor`)